qa/vagrant/src/test/resources/packaging/scripts/20_tar_package.bats

```diff
@@ -115,6 +115,24 @@ setup() {
     export ES_JAVA_OPTS=$es_java_opts
 }
 
+@test "[TAR]" start Elasticsearch with unquoted JSON option {
```

Comment:
This is fine, I wanted to do something like this the other day too but ultimately passed on it. I think it shows a shortcoming of our rest tests. We need the ability to start up a node with arbitrary JVM options/settings, execute requests and have them match a pattern. Bashing them into the bats tests is the best we have right now.